infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Orchestrator.java

```diff
@@ -264,6 +264,7 @@ public void shutdown() {
             log.debug("Orchestrator executor awaitTermination interrupted : {}", ie);
             Thread.currentThread().interrupt();
         }
+        getRuntime.cleanup(CorfuRuntime::shutdown);
```

Comment:
The management server already shuts this down. It shuts down the services using this first i.e., the orchestrator and the management service and then finally the runtime.
Shutting this down here will only create a new instance by the management service if that hasn't been shut down yet.

Reply:
Done.